tests/test_api.py

```diff
@@ -400,7 +426,6 @@ def test_no_subsection_grade(self, mock_graded_subsections):
             row = rows[i].split(',')
             assert row[grade_column_index] == '1'
 
-
```

Comment:
i don't care about this but does quality not complain?

Reply:
I don't think it enforces spacing between functions (only classes?). Either way, there was an extra newline here and I got rid of it.